packages/obonode/obojobo-chunks-iframe/converter.js

```diff
@@ -23,13 +23,14 @@ const slateToObo = node => ({
 		height: node.content.height,
 		initialZoom: node.content.initialZoom,
 		autoload: node.content.autoload,
-		controls: node.content.controls
+		controls: node.content.controls,
+        size: node.content.size
```

Comment:
Pretty minor, but this line is indented using spaces while the rest of the file uses tabs.